### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by kubaaa203** [MeteorDevelopment/meteor-client#4066]
### Describe the bug

ESP module makes/shows items that player is holding very weirdly

![vaizdas_2023-09-12_211805339](https://github.com/MeteorDevelopment/meteor-client/assets/86421878/30127353-3c67-49ff-aa8b-f6e0494b900d)

### Steps to reproduce

```bash
turn on ESP
```

### Meteor Version

0.5.4 1932

### Minecraft Version

1.16.1

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported (I have searched bug reports on GitHub).

- [X] This is a valid bug (I am able to reproduce this on the latest dev build).

**Comment by Big-Iron-Cheems** [comment]
By any chance, are you using Iris as well?

**Comment by kubaaa203** [comment]
Yes, is that the problem? I am using iris because meteor doesn't have built-in optifine in it and its's not compatable with it. Are there alternatives for Iris for shaders?

**Comment by Fiz-Victor** [comment]
unfortunately nope

**Comment by Big-Iron-Cheems** [comment]
This is a known bug we yet have to fix sadly. 
It should only happen in the Shader mode of ESP from what I recall.
You can keep using Iris obviously, since you need it to load shaders etc. on Fabric.

**Comment by pizzav-xyz** [comment]
> This is a known bug we yet have to fix sadly. It should only happen in the Shader mode of ESP from what I recall. You can keep using Iris obviously, since you need it to load shaders etc. on Fabric.

@Big-Iron-Cheems why still not fixed?
